jquery.floatThead.js

```diff
@@ -648,10 +662,22 @@
             if (tableContainerGap > scrollingContainerTop || scrollingContainerTop - tableContainerGap > tableHeight) {
               top = tableOffset.top - windowTop;
               unfloat();
+              // if we scrolled horizontally, table size may have changed
+              // for example, if there's padding to left of table and when scroll all the way to left padding "reappears"
+              setFloatWidth(); 
               triggerFloatEvent(false);
             } else {
               top = tableOffset.top + scrollingContainerTop  - windowTop - tableContainerGap;
+              // when scrolling horizontally within a modal, size of table may have increased
+              // for example, if there's padding to left of table and when scroll padding is no longer visible
+              setFloatWidth(); 
```

Comment:
I am not sure I am ok with calling this expensive method on every scroll event. this will be slow for older browsers / phones